src/main/webapp/script.js

```diff
@@ -22,19 +22,34 @@ function getComments() {
       
       // Display the maximum number of comments
       for (let i = 0; i < data.length; i++) {
-          // Format i comments in a list and display to the webpage.
-          if (data[i] != "") {
-              comments.appendChild(createListElement(data[i]));
-            }
+        // Format i comments in a list and display to the webpage.
+        if(i%3 == 0) {
+          // Add a return between every comment
+          if (i != 0) {
+            comments.appendChild(createListElement("\n"));
+          }
+          // Bold only the names with each comment
+          comments.appendChild(createBoldElement(data[i]));
+          }
+        else if (data[i] != "") {
+          comments.appendChild(createListElement(data[i]));
         }
+      }
     });
 }
 
-/** Create an <li> element containing text.*/
+/** Create a <li> element containing text.*/
 function createListElement(text) {
-    const liElement = document.createElement('p');
-    liElement.innerText = text;
-    return liElement;
+  const liElement = document.createElement('p');
```

Comment:
2 space or 4 space for indents?

Reply:
2 space seems to match with the rest of the file!

Reply:
2 space seems to match the rest of the file!